scripts/xtensa-build-all.sh

```diff
@@ -252,7 +250,7 @@ do
 			# default key for TGL
 			if [ -z "$PRIVATE_KEY_OPTION" ]
 			then
-				PRIVATE_KEY_OPTION="-D${SIGNING_TOOL}_PRIVATE_KEY=$pwd/keys/otc_private_key_3k.pem"
+	PRIVATE_KEY_OPTION="-D${SIGNING_TOOL}_PRIVATE_KEY=$SOF_TOP/keys/otc_private_key_3k.pem"
```

Comment:
Some indent issue here, or you did this on purpose?  and also line 268. maybe mixed use of space and tab

Reply:
I did this on purpose to make the code visible again. I could split it on two lines if you insist but it would probably not look much better.